backend/src/test/java/com/library/backend/model/MealTest.java

```diff
@@ -0,0 +1,12 @@
+package com.library.backend.model;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+class MealTest {
```

Comment:
What is this class for?

I don´t see a reason for it.
If you don´t use it, please delete it! :-)